commit c3b20dc42de36fc7b5b1a3a572e76b034d4d2932 (refs/changes/97/2997/3)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-11-25T00:51:43+00:00 (1 year, 10 months ago)
    
    VOLAPI-96 sdcadm post-setup volapi errors when default channel is not "dev"
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/lib/post-setup/volapi.js b/lib/post-setup/volapi.js
index 504c996..39aab97 100644
--- a/lib/post-setup/volapi.js
+++ b/lib/post-setup/volapi.js
@@ -250,8 +250,7 @@ function do_volapi(subcmd, opts, args, cb) {
             proc.execute({
                 sdcadm: self.sdcadm,
                 log: self.log,
-                progress: self.progress,
-                source: 'https://updates.joyent.com'
+                progress: self.progress
             }, next);
         },
 
